src/Zipper.ts

```diff
@@ -101,6 +101,12 @@ export const length: <A>(fa: Zipper<A>) => number = (fa) => fa.lefts.length + 1
  */
 export const toArray: <A>(fa: Zipper<A>) => Array<A> = (fa) => A.snoc(fa.lefts, fa.focus).concat(fa.rights)
 
+/**
+ * @category destructors
+ * @since 0.1.6
```

Comment:
I guess this should belong to a new version, `0.1.7`. Am I right @gcanti?

Reply:
This was generated automatically. I can change that to whatever version it should be.

Reply:
current version is `0.1.21` so it will be `0.1.22`